database/seeders/InvoicesTableSeeder.php

```diff
@@ -17,7 +17,7 @@ public function run()
         Invoice::create([
             'order_id'                  => 1,
             'admin_id'                  => 1,
-            'time_issued'               => '2020-07-13 06:13:17',
+            'time_issued'               => now(), 
```

Comment:
Making the purchase today (when you run the seeder) so it's always on the graph.